content/general/concepts/virtual-machines/virtual-machines.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: 'Virtual Machines'
+Description: 'Virtual machines are virtual versions of physical computers.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Virtual Machines'
+  - 'Cloud Computing'
+  - 'Security'
+  - 'DevOps'
+CatalogContent:
+  - 'foundations-of-cloud-computing'
+  - 'paths/computer-science'
+---
+
+A virtual machine, or VM, is a virtual version of a physical computer. VMs can perform the same computing functions as physical computers, such as running applications or storing data. However, VMs do not use physical hardware to perform these tasks. Instead, VMs use virtual, software-defined versions of hardware that are installed on a host server. VMs only exist as code.
+
+A single server or physical computer can run multiple VMs at the same time. Each VM has its own operating system and functions as an independent compute resource, even though they may share the same host server. For example, if you had a computer with macOS, you could use a VM to run a separate instance of the Windows operating system on top of your existing macOS computer.
+
+## How do virtual machines work?
+
+Virtual machines run on top of the operating system of the host machine, similar to other applications on a physical computer. All VMs on a single host share the same physical resources, such as memory and storage.
+
+Multiple VMs can be managed with virtual machine software, also called a “hypervisor”. A hypervisor is used to create, manage, and run the VMs on a host machine. The hypervisor is used to manage hardware resources from the host machine, which enables these resources to be shared amongst multiple VMs.
+
+## Benefits of using virtual machines
+
+Virtual machines can provide many benefits, such as:
+
+### Improved hardware utilization
```

Comment:
```suggestion
### Improved Hardware Utilization
```

For Docs entries, even the subheaders use "Title Case". Can we apply this edit to the remainder of the file?